3-platform/shopping-cart/app/src/main/java/com/_6core/platform/shopping/cart/app/validator/ValidationHandler.java

```diff
@@ -0,0 +1,37 @@
+package com._6core.platform.shopping.cart.app.validator;
+
+public abstract class ValidationHandler<T> implements Validator<T> {
```

Comment:
Is validationHandler eq to Validator? or it is completely new class that responsible for building validators chain?

Reply:
ValidatorHandler responsible for building validators chain. I will rename this to ValidationChainBuilder.